dbt/README.md

```diff
@@ -348,27 +348,59 @@ names of tables and views in Athena
 In addition to database namespacing, views should be named with a `vw_` prefix
 (e.g. `location.vw_pin10_location`) to mark them as a view, while tables do not
 require any prefix (e.g. `location.tax`).
+
 Finally, for the sake of consistency and ease of interpretation, all tables and views should be named using the singular case e.g. `location.tax` rather than `location.taxes`.
 
+### Model location
+
+Models are generally defined in the `schema.yml` file within each database
+subdirectory. Resources related to each model should be defined inline (with
+the exception of columns, see [Column descriptions](#column-descriptions).
+
+For complicated models with *many* columns or tests, we split `schema.yml`
+files into individual files per model. These files should be contained in a
+`schema/` subdirectory within each database directory, and should be named
+using the fully namespaced model name. For example, the model definition
+for `iasworld.sales` lives in `models/iasworld/schema/iasworld.sales.yml`.
+
 ### Model description
 
 All new models should include, at minimum, a
 [description](https://docs.getdbt.com/reference/resource-properties/description)
-of the model itself. We use the following pattern to determine where to
-define the Markdown text that comprises these descriptions:
+of the model itself. We store these model-level descriptions as [docs
+blocks](https://docs.getdbt.com/reference/resource-properties/description#use-a-docs-block-in-a-description)
+within the `docs.md` file of each schema subdirectory.
```

Comment:
**[Question, non-blocking]** Do we want to continue storing model-level descriptions as docs blocks? It seems less likely that a description used by a model will be shared by another entity. Is the reasoning here that model descriptions are likely to be very long, so we want to store them outside the schema file to avoid clutter?